API.pm

```diff
@@ -445,14 +445,16 @@ sub getPublicPlaylists {
 
 	my $args = {
 		type  => $type =~ /(?:last-created|editor-picks)/ ? $type : 'editor-picks',
-		limit => 100,		# for whatever reason this query doesn't accept more than 100 results
-		_ttl  => QOBUZ_EDITORIAL_EXPIRY,
+		limit => QOBUZ_USERDATA_LIMIT,
+		_ttl  => QOBUZ_USER_DATA_EXPIRY, #QOBUZ_EDITORIAL_EXPIRY,
```

Comment:
I believe this change is unnecessary. That Editorial data doesn't change every minute. I as a user probably wouldn't even notice when it changes. I'd be surprised if it changed more than once a day?

Reply:
That makes sense. I wasn't quite sure about that yet and therefore left the old value commented out (at least in my code :-)). I then set it back to 'QOBUZ_EDITORIAL_EXPIRY'.
But 'limit' has to be changed, otherwise even with _pagingGet() there will only be a hundred entries.
-
Das macht Sinn. ich war mir da noch nicht ganz sicher und hatte deshalb den alten Wert noch auskommentiert dahinter stehen lassen (zumindest in meinem Kode :-)).  Ich setze es dann wieder auf 'QOBUZ_EDITORIAL_EXPIRY'.
'limit' muss aber geändert werden, sonst kommen auch mit _pagingGet() nur hundert Einträge.